README.md

```diff
@@ -13,11 +13,11 @@ generates.
 Magic Modules generates GCP support for:
 
 * Terraform
-* Ansible
 
 In addition, Magic Modules generates support for several companion
 features/tools:
 
+* Terraform Validator
 * Terraform Google Inventory Mapper
```

Comment:
This is TGC, we can probably remove it as well? (the name was slightly obfuscated at the time this was written)

Reply:
done